components/count-badge/count-badge-icon.js

```diff
@@ -75,6 +75,26 @@ class CountBadgeIcon extends CountBadgeMixin(LitElement) {
 		this._badgeId = getUniqueId();
 	}
 
+	connectedCallback() {
+		super.connectedCallback();
+		const self = this;
```

Comment:
Binding the mutation function didn't seem to work, as I kept getting a "this is undefined" error. So I just created a self variable to use in the mutation listener.